### PR TITLE
fix(trello_importer): tg-4782 Trello importer generate empty attachments

### PR DESCRIPTION
**Pull request by bameda**
Related to https://tree.taiga.io/project/taiga/issue/4782

## How to validate:
- [x] Review the code
- [x] Test: 
  - Create a project in trello with cards that contains attachments.
  - Import the project in Taiga
  - Check the content of the imported attachments. For example, you should see the images in the kanban (with max zoom level)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## 6.5.0 (unreleased)
 
 - Now member limits by projects are counted by owner and not by project.
+- Trello importer generate empty attachments (issue #tg-4782)
```
Comment (yamila-moreno) [inline CHANGELOG.md]: I'd say this is a fix

File: taiga/importers/trello/importer.py [hunk 2]
```diff
@@ -52,8 +52,23 @@ def __init__(self, api_key, api_secret, token):
         else:
             self.oauth = None
 
+
```
Comment (daniel-herrero) [inline taiga/importers/trello/importer.py]: Better one blank line, not two

File: taiga/importers/trello/importer.py [hunk 3]
```diff
@@ -52,8 +52,23 @@ def __init__(self, api_key, api_secret, token):
...
+        if response.status_code == 401:
+            raise exc.AuthenticationFailed(_("Unauthorized: %s at %s") % (response.text, response.url))
+        if response.status_code == 403:
+            raise exc.PermissionDenied(_("Unauthorized: %s at %s") % (response.text, response.url))
+        if response.status_code == 404:
+            raise exc.NotFound(_("Resource Unavailable: %s at %s") % (response.text, response.url))
+        if response.status_code != 200:
+            raise exc.WrongArguments(_("Resource Unavailable: %s at %s") % (response.text, response.url))
+
+
```
Comment (daniel-herrero) [inline taiga/importers/trello/importer.py]: Same as above, it's better with a single blank line

**Review by daniel-herrero — APPROVED** [review]
Well done! Hi five!

**Pull request merged**